src/AppBundle/Validator/Constraints/RecaptchaValidator.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/*
+ * This file is part of the Sententiaregum project.
+ *
+ * (c) Maximilian Bosch <maximilian.bosch.27@gmail.com>
+ * (c) Ben Bieler <benjaminbieler2014@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace AppBundle\Validator\Constraints;
+
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+
+/**
+ * Class RecaptchaValidator.
+ *
+ * @author Benjamin Bieler <benjaminbieler2014@gmail.com>
+ */
+class RecaptchaValidator extends ConstraintValidator
+{
+    /**
+     * Constructor.
+     *
+     * @param Container $container
+     */
+    public function __construct(Container $container)
+    {
+        $this->container = $container;
```

Comment:
it's a best practice here to define every property in PHP in class scope before using them in methods (private $dependency at the top for instance)